relevanceai/clusterer/clusterer.py

```diff
@@ -23,9 +23,16 @@
 
 import numpy as np
 
-from typing import Union, List, Dict, Optional
+from scipy.spatial import distance
```

Comment:
Move this to inside the function. People shouldn't need `scipy` to run a function